test/java/org/opendatakit/briefcase/export/ExportFormsTest.java

```diff
@@ -124,8 +124,6 @@ public void knows_if_all_selected_forms_have_a_valid_configuration() {
     FormStatus form = forms.get(0);
     form.setSelected(true);
 
-    assertThat(forms.allSelectedFormsHaveConfiguration(), is(false));
```

Comment:
I think that we should get back this assertion as a consequence of previous comments